### DISCUSSION TITLE
ODS Viewer for VSCode?

### DISCUSSION THREAD

**Discussion by QuitStalin** [fabrizioschiavi/pragmatapro-semiotics#7] [Q&A]
Hello! I've been scrolling on GItHub for some projects to contribute to and stumbled into this repo. I've never seen an .ods file and want to know if anyone here knows of any VSCode extension that lets you preview or edit .ods files?

**Comment by amake** [comment]
ODS files are usually edited with [LibreOffice](https://www.libreoffice.org/). I don't use VSCode so I can't help there.

**Reply by QuitStalin** [reply]
Thanks for the reply I'll have to check that out.
